cmd/cmd.go

```diff
@@ -112,8 +126,23 @@ func (o *RootCommandOption) Run(streams IOStreams) error {
 	}
 	fmt.Fprintf(streams.Out, "Load template from %q directory\n", cnf.Template)
 
-	if _, err := os.Stat(o.outDir); os.IsNotExist(err) {
-		err := os.Mkdir(o.outDir, 0755)
+	outDir := defaultOutput
+	outFilename := ""
+
+	if o.output != defaultOutput {
+		outDir = o.output
+		if isSpecifiedOutputFilename {
+			splitOutput := strings.Split(o.output, "/")
```

Comment:
`filepath.Split(t)` is more suitable function. It could split base-directory and filename at once :)